test/scripts/ers_python_publisher_test

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+import erskafka.ERSPublisher as erspub
+from erskafka.ERSPublisher import ERSException, SeverityLevel  # import the custom exception and the SeverityLevel enum
+from time import sleep
+import click
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--kafka-address', type=click.STRING, default="monkafka.cern.ch", help="Address of the Kafka broker.")
+@click.option('--kafka-port', type=click.INT, default=30092, help='Port of the Kafka broker.')
+@click.option('--number-of-issues', type=click.INT, default=10, help='Number of issues to send.')
+
+def cli(kafka_address, kafka_port, number_of_issues):
+    bootstrap = f"{kafka_address}:{kafka_port}"
+    conf = {"bootstrap": bootstrap, "topic": "monitoring.ers_stream"}
```

Comment:
This topic is a bit misleading. As it is written, this will create a topic that is 
`monitoring.monitoring.ers_stream`. Which is not necessarily wrong, but it will be ignored by the microservice that puts the issue in the database. 

I understand the need for a non-disturbing topic for tests, I suggest this. 
 - Add an option with the topic name.
 - Default the option to `ers_stream` so that the default goes inside the database